bin/generate_unfiltered_sce.R

```diff
@@ -51,6 +57,16 @@ if(!(stringr::str_ends(opt$unfiltered_file, ".rds"))){
   stop("unfiltered file name must end in .rds")
 }
 
+# check that mitochondrial gene list exists
+if(!file.exists(opt$mito_file)){
+  stop("Mitochondrial gene list is required using -m or --mito_file")
+}
+
+# read in mitochondrial gene list
+mito_genes <- readr::read_tsv(opt$mito_file, col_names = "gene_id") %>%
+  dplyr::pull("gene_id") %>%
+  unique()
```

Comment:
If this is just a file with one line per row, we can simplify this a bunch:
```suggestion
mito_genes <- unique(scan(opt$mito_file, what = "character"))
```